test/unit/lib/plugins/aws/package/compile/events/cognito-user-pool.test.js

```diff
@@ -803,3 +952,437 @@ describe('AwsCompileCognitoUserPoolEvents', () => {
     });
   });
 });
+
+describe('AwsCompileCognitoUserPoolEvents - runServerless', () => {
```

Comment:
Sorry I've missed that earlier, let's follow the convention from other test files where we name the top describe with the path to the test file